analytics-core/src/main/java/com/segment/analytics/ValueMap.java

```diff
@@ -51,6 +51,10 @@ public ValueMap() {
     delegate = new LinkedHashMap<>();
   }
 
+  public ValueMap(int initialCapacity) {
```

Comment:
Is there any value in making this `public` over `protected` or package-private? Callers wouldn't really create instances of this directly. Those who really need the int overload, can create their own pre-sized maps and use the map constructor.